compliance_checker/tests/test_ioos_profile.py

```diff
@@ -707,10 +707,14 @@ def test_check_ioos_ingest(self):
         ds.setncattr("ioos_ingest", "false")
         self.assertTrue(self.ioos.check_ioos_ingest(ds).value)
 
-        # value anything but false
+        # value true
         ds.setncattr("ioos_ingest", "true")
-        self.assertFalse(self.ioos.check_ioos_ingest(ds).value)
+        self.assertTrue(self.ioos.check_ioos_ingest(ds).value)
+
+        # anything else should fail
         ds.setncattr("ioos_ingest", 0)
         self.assertFalse(self.ioos.check_ioos_ingest(ds).value)
+        ds.setncattr("ioos_ingest", "True")
```

Comment:
So to be sure, this attribute is case sensitive?

Reply:
The spec does say "false":

https://ioos.github.io/ioos-metadata/ioos-metadata-profile-v1-2.html#ioos-ingest

I thought about making it case-insensitive - @mwengren, any input on this?